code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1109,39 +1103,43 @@
 	return TRUE
 
 /datum/species/proc/apply_damage(damage, damagetype = BRUTE, def_zone = null, blocked, mob/living/carbon/human/H)
-	blocked = (100-(blocked+armor))/100
-	if(!damage || blocked <= 0)
+	var/hit_percent = (100-(blocked+armor))/100
```

Comment:
Changing the value of one of the argument was an accident waiting to happen. We now use a different var and name.